drivers/timer/sam0_rtc_timer.c

```diff
@@ -20,7 +20,7 @@
 #include <sys_clock.h>
 
 /* RTC registers. */
-#define RTC0 ((RtcMode0 *) DT_RTC_SAM0_BASE_ADDRESS)
+#define RTC0 (&(RTC->MODE0))
```

Comment:
Please keep using the base address from dts rather than from ASF. We are doing this by convention for all the vendors

Reply:
ACK, changed.